python-3.10/Dockerfile

```diff
@@ -105,30 +105,29 @@ RUN mkdir /home/default \
         sympy \
         tabulate \
         tomlkit \
-        torch \
         tqdm \
         vincent \
         wheel \
         xgboost \
         xlrd \
-	&& chown :users -R /home/default \
-	&& chmod a+rwx -R /home/default
-
-RUN pip3 install --no-cache-dir --upgrade --force-reinstall git+https://github.com/keboola/python-docker-application.git@2.2.0 \
+    && pip3 install --no-cache-dir --upgrade --force-reinstall git+https://github.com/keboola/python-docker-application.git@2.2.0 \
     && pip3 install --no-cache-dir --find-links https://h2o-release.s3.amazonaws.com/h2o/latest_stable_Py.html h2o \
     && pip3 install --no-cache-dir --upgrade --force-reinstall \
         git+https://github.com/keboola/sapi-python-client.git@0.4.0 \
         keboola.component \
-        cryptography\<37 \
+        charset-normalizer\<3 \
+        cryptography\<41 \
+        pytz\<2023 \
```

Comment:
These packages are new, is it OK?

Reply:
Yes, these are required so that the pip check passes, they are requirements of the snowflake python connector.

Reply:
and it's a new dependency? or why it was not required before?

Reply:
it's been off and on for years.  sometimes they're required and sometimes not.  It really depends on release timings.